ramp-database/ramp_database/tools/tests/test_event.py

```diff
@@ -263,6 +272,68 @@ def test_check_event(session_scope_function):
     assert len(event) == 1
 
 
+def test_delete_event(session_scope_function):
+    # add sample problem
+    problem_name = 'iris'
+    ramp_config = read_config(ramp_config_iris())
+
+    internal_ramp_config = generate_ramp_config(ramp_config)
+    setup_ramp_kit_ramp_data(internal_ramp_config, problem_name, depth=1)
+    add_problem(session_scope_function, problem_name,
+                internal_ramp_config['ramp_kit_dir'],
+                internal_ramp_config['ramp_data_dir'])
+
+    # add sample event
+    add_event(session_scope_function, problem_name,
+              internal_ramp_config['event_name'],
+              internal_ramp_config['event_title'],
+              internal_ramp_config['sandbox_name'],
+              internal_ramp_config['ramp_submissions_dir'],
+              is_public=True, force=False)
+
+    event = get_event(session_scope_function,
+                      internal_ramp_config['event_name'])
+    assert event
+
+    # add event-team
+    event_name, username = internal_ramp_config['event_name'], 'test_user'
+    sign_up_team(session_scope_function, event_name, username)
+    event_team = get_event_team_by_name(session_scope_function,
+                                        event_name, username)
+    assert event_team
+
+    # add event admin
+    add_event_admin(session_scope_function, event_name, username)
+    assert get_event_admin(session_scope_function, event_name, username)
+
+    # check if the event is connected to any score type in the database
+    event_score_types = get_score_type_by_event(session_scope_function, event)
+    assert len(event_score_types) > 0
+
+    # check if the event is connected to any cv_fold in the database
+    event_cv_fold = get_cv_fold_by_event(session_scope_function, event)
+    assert len(event_cv_fold) > 0
+
+    # add the submission to the event
+    from ramp_database.tools.submission import get_submission_by_id
+    submission = get_submission_by_id(session_scope_function, event.id)
+    assert submission
+
+    delete_event(session_scope_function, internal_ramp_config['event_name'])
+    # make sure event and all the connections were deleted
+    event_test = get_event(session_scope_function, None)
+    assert len(event_test) == 0
+    assert not get_event_team_by_name(session_scope_function,
+                                      event_name, username)
+    assert not get_event_admin(session_scope_function, event_name, username)
+    event_score_types = get_score_type_by_event(session_scope_function, event)
+    assert len(event_score_types) == 0
+    event_cv_fold = get_cv_fold_by_event(session_scope_function, event)
+    assert len(event_cv_fold) == 0
+    with pytest.raises(AttributeError):
```

Comment:
I would instead my the query finishing with all() and check that we have an empty list

Reply:
coming back to your other comment about not writing the get_ queries: would you write it the the test_event.py itself?